infrastructure/terraform/ingress.yaml

```diff
@@ -18,10 +18,6 @@ spec:
           backend:
             serviceName: taskcluster-ui
             servicePort: 80
-        - path: '/docs'
-          backend:
-            serviceName: taskcluster-ui
-            servicePort: 80
```

Comment:
This will get handled by the `/` path above

Reply:
I guess that will happen after Hassan's work is merged?

Reply:
It should be the case already -- everything that's not matched by a more specific route goes to taskcluster-ui, so there's no need to specifically list `/docs` as going to taskcluster-ui.

Reply:
hm but do we have that path in netlify config then?

Reply:
anyways, I approved - we can always go back, it's not a big deal

Reply:
Actually, I just tried deploying with this change - docs don't seem to work for me. Have you tried deploying this on your cluster?